packages/jv-ui-components/material-ui/MultiSelect/MultiSelect.tsx

```diff
@@ -58,21 +67,21 @@ export const MultiSelect = forwardRef((props: any, ref) => {
         renderValue={(value: any[]) => value.join(", ")}
         id={id}
         labelId={id + "_label"}
-        className={className + " jv-mInput-select jv-mInput-text mui"}
+        className={className + " jv-mInput-select mui"}
         multiple
         variant="outlined"
         input={<MuiOutlinedInput label="Tag" />}
-        slotProps={{
-          elevation: 4,
-          style: {
-            maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
-            width: 250,
-          },
-        }}
         MenuProps={{
           classes: {
             list: "jv-mChecklist jv-mChecklistLarge mui",
           },
+          PaperProps: {
```

Comment:
if I remember correctly, this `PaperProps` has been deprecated...
isn't `slotProps` the property to use ?

Reply:
This is what was in storybook but i can revert it.